src/components/Garden/Feed/ProposalCreator.tsx

```diff
@@ -1,24 +1,25 @@
 import React, { useEffect, useState } from 'react'
-import {
-  EthIdenticon,
-  GU,
-  shortenAddress,
-  textStyle,
-  useTheme,
-} from '@1hive/1hive-ui'
+import { GU, shortenAddress, textStyle, useTheme } from '@1hive/1hive-ui'
 import ProposalIcon from '@components/ProposalIcon'
 
 import { convertToString } from '@/types'
 import { getProfileForAccount } from '@lib/profile'
 import { dateFormat } from '@utils/date-utils'
 import { addressesEqual } from '@utils/web3-utils'
 import { ZERO_ADDR } from '@/constants'
+import { ProposalType } from '@/hooks/constants'
```

Comment:
i would recommend to move the `constants` file out from the hooks since are types that are being used across the app and rename it to something like `types`

Reply:
Sure, I can do that.

Reply:
Updated!